src/api/CoinBase.ts

```diff
@@ -0,0 +1,27 @@
+import { CoinBaseResponse, SourceApi } from '../types/RelayPricer';
+import fetch, { Response } from 'node-fetch';
+
+export default class CoinBase implements SourceApi {
+    // we should get this address from config file or enviroments
+    private readonly URL: string = 'https://api.coinbase.com/v2/exchange-rates';
+
+    async query(input: string, output: string): Promise<number> {
+        output = output.toUpperCase();
```

Comment:
better to create another variable, const upperCaseOutput, instead of mutating the same variable.

Reply:
I wasn't sure about this. I'll be updating it.

Reply:
done